readthedocs-theme/templates/readthedocs/pricing.html

```diff
@@ -90,7 +90,7 @@ <h2 class="ui header centered">
                 </div>
                 <div class="item">
                   <div class="content">
-                    <div class="header">2 Concurrent builds</div>
```

Comment:
The advanced plan has the same issue on "4 Concurrent builds"

Reply:
and Pro as well

Reply:
I made a new push applying the fix to Pro and Advanced.